web-quiz-app/src/components/HeaderStudent.jsx

```diff
@@ -38,13 +38,16 @@ function HeaderStudent() {
       </Dropdown.Toggle>
 
       <Dropdown.Menu>
-        <Dropdown.Item href="#/action-1">My Profile</Dropdown.Item>
-        <Dropdown.Item href="#/action-2">Switch Account</Dropdown.Item>
-        <Dropdown.Item href="#/action-3">Logout</Dropdown.Item>
+        <Dropdown.Item href="#/action-1" onClick={logoutUser}>Logout</Dropdown.Item>
       </Dropdown.Menu>
     </Dropdown>
   </nav>
   );
+
+  function logoutUser() {
+    localStorage.clear();
+    window.location.href = '/';
+}
```

Comment:
indentation for the bracket. 
```suggestion
  }
```